src/pyannote/audio/__main__.py

```diff
@@ -583,6 +655,12 @@ def benchmark(
         print(f"Could not load pretrained pipeline from {pipeline}.")
         raise typer.exit(code=1)
 
+    tags = getattr(pretrained_pipeline, "tags", [])
```

Comment:
In retrospect, I think it would be more readable to always compute every metrics (and simply use "empty" predictions when it does not exist in the output). What do you think? 

Same for the reference -- if not there, use empty reference.

Reply:
Or you could add two flags `--diarization` and `--transcription` and fail when, for instance, we ask for transcription and there is no reference for it. What do you think?